### ISSUE TITLE
(IMPORT FAILED): D:\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-IF_MemoAvatar；(IMPORT FAILED): D:\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-IF_AI_tools

### ISSUE THREAD

**Issue by 1696hh** [if-ai/ComfyUI-IF_MemoAvatar#12]
(IMPORT FAILED): D:\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-IF_MemoAvatar
(IMPORT FAILED): D:\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-IF_AI_tools
![2024-12-17_174507](https://github.com/user-attachments/assets/1ae613ef-7291-42c0-b0af-fcd82e5a8756)

**Comment by 1696hh** [comment]
It won't load at all. It keeps giving me errors.

**Comment by if-ai** [comment]
You are correct on the manager should be an update It was supposed to broadcast the update but it didn't please try again, thanks

**Comment by if-ai** [comment]
by the way don't use IF_AI tools yet you should get IF_LLM instead

**Comment by smithsun00** [comment]
When you see these load failure - look at the logs in the terminal/cmd - search for the package name or just something like - MemoAvatar - it will spit more information of what actually the reason is - for me it were a couple of some other packages needing to be updated, like triton and stuff.

**Comment by 1696hh** [comment]
> You are correct on the manager should be an update It was supposed to broadcast the update but it didn't please try again, thanks您对经理的看法是正确的，应该是更新。它应该广播更新，但没有，请重试，谢谢

This is how it looks after updating, every time I update, after rebooting, the problem still exists

**Comment by yangquanbiubiu** [comment]
Same error here. I can't import ComfyUI-IF_MemoAvatar even installed all packages mentioned in the requirements.txt.

**Comment by yangquanbiubiu** [comment]
Same error here. I can't import ComfyUI-IF_MemoAvatar even installed all packages mentioned in the requirements.txt.

**Comment by JPW0080** [comment]
Win 11 x64:

Command Prompt:
Activate ComfyUI ENV...

pip install moviepy==1.0.3

Download > https://github.com/modelscope/FunASR/archive/refs/heads/main.zip
Open the above ^ .zip

Extract > FunASR-main\funasr > \ComfyUI\custom_nodes\ComfyUI-IF_MemoAvatar\memo   #NOTE: Make sure the path is extracted as > \ComfyUI\custom_nodes\ComfyUI-IF_MemoAvatar\funasr

Go to >  \ComfyUI\custom_nodes\ComfyUI-IF_MemoAvatar\funasr\download
Create a copy of download_model_from_hub.py

Rename the above ^ copy as > download_from_hub.py

![memo-funasr-gradio-troubleshoot](https://github.com/user-attachments/assets/f2c038d5-1d52-4088-bdba-d8b30fb8cc91)


#NOTE: Before closing memo, remember to reinstall moviepy...

#Ex:

Command Prompt:

Activate ComfyUI ENV...

pip install moviepy==2.1.1
